src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -70,6 +94,10 @@ export default function BulkEmailForm(props) {
       emailData.append('send_to', JSON.stringify(selectedRecipients));
       emailData.append('subject', subject);
       emailData.append('message', editorRef.current.getContent());
+      if (isScheduled) {
+        const { scheduleDate, scheduleTime } = dateTime;
+        emailData.append('schedule', new Date(`${scheduleDate} ${scheduleTime}`).toISOString());
+      }
```

Comment:
Can we also make sure to capture and pass along the browser timezone? This will be used as a fallback option if the user doesn't have a "preferred timezone" set in their account settings.

The backend is expecting checking for a key named `browser_timezone` in the data: https://github.com/openedx/edx-platform/blob/f2582c4e9e6664fd77ee8cfe648b40558962064a/lms/djangoapps/instructor/views/api.py#L2709.

Reply:
The time zone only really matters to scheduled requests (at this time) so I think it's fine to skip sending that data on non-scheduled requests.